pkg/strategy/all_in_one_test.go

```diff
@@ -6,6 +6,8 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/jaegertracing/jaeger-operator/pkg/util"
```

Comment:
Import is out of order :/

Reply:
`goimports` seems to be buggy. I will fix it manually.